robosat/tools/download.py

```diff
@@ -8,29 +8,42 @@
 from PIL import Image
 from tqdm import tqdm
 
-from robosat.tiles import tiles_from_csv, fetch_image
+from robosat.tiles import tiles_from_csv, fetch_image, tile_to_bbox
+from robosat.utils import leaflet
+from robosat.log import Log
 
 
 def add_parser(subparser):
     parser = subparser.add_parser(
-        "download", help="downloads images from Mapbox Maps API", formatter_class=argparse.ArgumentDefaultsHelpFormatter
+        "download", help="downloads images from a remote server", formatter_class=argparse.ArgumentDefaultsHelpFormatter
     )
 
-    parser.add_argument("url", type=str, help="endpoint with {z}/{x}/{y} variables to fetch image tiles from")
+    parser.add_argument(
+        "url", type=str, help="endpoint with {z}/{x}/{y} or {xmin},{ymin},{xmax},{ymax} variables to fetch image tiles"
+    )
     parser.add_argument("--ext", type=str, default="webp", help="file format to save images in")
     parser.add_argument("--rate", type=int, default=10, help="rate limit in max. requests per second")
+    parser.add_argument("--type", type=str, default="XYZ", help="service type to use (e.g: XYZ, WMS or TMS)")
```

Comment:
Maybe add `choices`; then register handlers for the choices and have them download the corresponding files?